### PR TITLE
Add blog post about Vitess graduation

### PR DESCRIPTION
**Pull request by avaidyanatha**
## Main Changes
- Title is self explanatory, add interview post where we get Sugu's thoughts about the graduation process and describe the Vitess community's efforts to push through the milestone!

### CODE REVIEW

File: content/en/blog/2020-01-06-vitess-graduation-retrospective.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+author: "Abhi Vaidyanatha"
+date: 2020-01-06T09:07:21-08:00
+slug: "2020-01-06-vitess-graduation-retrospective"
+tags: ['graduation', 'vitess graduation']
+title: "Vitess Graduation Retrospective"
+---
+
+Last week, Vitess became the eighth CNCF project to reach graduation joining a host of amazing projects such as Kubernetes, Prometheus, Envoy, CoreDNS, containerd, Fluentd, and Jaeger. To contextualize this milestone, I picked some tidbits from Sugu’s brain, allowing him to share internal perspective about how we got here, and some of the hurdles along the way.
```
Comment (deepthi) [inline content/en/blog/2020-01-06-vitess-graduation-retrospective.md]: The date of graduation was Nov 5, 2019 (if it matters). `Last week` seems misleading.

File: content/en/blog/2020-01-06-vitess-graduation-retrospective.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+### Incubation
+
+Considering that it was only last February when the CNCF agreed to host Vitess as an incubating project, Vitess has made efficient strides toward becoming the de-facto standard for storage on Kubernetes. While I could talk more about our progress as a project, I’ll let these side-by-side quotes from Michael Demmer, one of Slack’s principal engineers, speak for themselves.
```
Comment (deepthi) [inline content/en/blog/2020-01-06-vitess-graduation-retrospective.md]: February 2018

File: content/en/blog/2020-01-06-vitess-graduation-retrospective.md (hunk 1 above)
Comment (morgo) [inline content/en/blog/2020-01-06-vitess-graduation-retrospective.md]: This will need updating. "In November.."

**Review by morgo — COMMENTED** [review]
LGTM, one comment.

File: content/en/blog/2020-01-06-vitess-graduation-retrospective.md (hunk 2 above)
Comment (deepthi) [reply content/en/blog/2020-01-06-vitess-graduation-retrospective.md]: Whose voice is this supposed to be? Blog author or Sugu?

**Review by deepthi — COMMENTED** [review]
Nice work.

File: content/en/blog/2020-01-06-vitess-graduation-retrospective.md (hunk 2 above)
Comment (avaidyanatha) [reply content/en/blog/2020-01-06-vitess-graduation-retrospective.md]: It's supposed to be my voice there, is it unclear?

Comment (deepthi) [reply content/en/blog/2020-01-06-vitess-graduation-retrospective.md]: cool, that's fine

**Review by deepthi — APPROVED** [review]
LGTM

**Review by deepthi — COMMENTED** [review]
Sorry, one more request.
Can you change `I` to `we`? There are two instances of that.

**Pull request merged**